selenium/testcontainers/selenium/__init__.py

```diff
@@ -60,9 +61,12 @@ def _configure(self) -> None:
 
     @wait_container_is_ready(urllib3.exceptions.HTTPError)
     def _connect(self) -> webdriver.Remote:
+        options = Options()
+        for key, value in self.capabilities.items():
+            options.set_capability(key, value)
```

Comment:
The `get_requirements.py` script pulled in the new Selenium dependency, that had breaking changes. We don't have a breaking change in the Testcontainers API though this, but if users use the transitive Selenium dependency in their tests, it can be breaking change for them.

Reply:
this is using chrome options unconditionally, even though the library support both - i used a base class in my fix for the same in #392

Reply:
You are right, great catch @alexanderankin.
I could not apply your change 1:1, because of the implicitly bumped dependency version, but will try to model it after your approach 👍

Reply:
What is interesting though, that the test we have using Firefox *does not fail*, do you have an idea why?

Reply:
Yeah if you read the source code of the selenium python package it seems like not too many differences - I forget what they are now but it seemed like a code smell to just use one even though it works. Probably safer in terms of future updates too

Reply:
It communicates intent better too, opting into common behavior but opting out of browser specific behavior